plugins/rest/komand_rest/actions/post/action.py

```diff
@@ -12,11 +29,16 @@ def __init__(self):
         )
 
     def run(self, params={}):
+        headers = params.get(Input.HEADERS, {})
+        body = params.get(Input.BODY, {})
+
+        convert_body_for_urlencoded(headers, body)
```

Comment:
```suggestion
        body = convert_body_for_urlencoded(headers, body)
```
Otherwise the object `body` inside `run` method context won't be affected